src/components/mac-titlebar.tsx

```diff
@@ -8,15 +9,20 @@ export default function MacTitlebar( {
 	className?: string;
 	children?: React.ReactNode;
 } ) {
+	const isFullscreen = useFullscreen();
+	const isRtl = isWindowFrameRtl();
+
 	return (
 		<div
 			className={ cx(
-				// Leave space for window controls depending on which side they are on
-				// Take into account the "chrome" padding, the position of which depends on the language direction
-				isWindowFrameRtl() &&
+				'transition-[padding] duration-700 ease-in-out',
```

Comment:
```suggestion
				'transition-[padding] duration-500 ease-in-out',
```
I would speed up this animation slightly. 500ms matches the sidebar transition